tuxemon/core/components/player.py

```diff
@@ -715,11 +714,16 @@ def get_adjacent_tiles(self, curr_loc, game):
 
 
 class Npc(Player):
-    def __init__(self, sprite_name="maple", name="Maple"):
-        # Initialize the parent menu class's default shit
-        Player.__init__(self, sprite_name, name)
+    def __init__(self, sprite_name="maple", slug="npc_maple"):
+        npcs = db.JSONDatabase()
+        npcs.load("npc")
+        npc_data = npcs['npc'][slug]

```

Comment:
I get an error from this line of code because `npcs` is a `JSONDatabase()` object, not a dictionary, so doing `npcs['npc']` will not work:

```python
Traceback (most recent call last):
  File "/tmp/Tuxemon/tuxemon/core/components/event/__init__.py", line 169, in execute_action
    self.actions[action.type]["method"](game, action)
  File "/tmp/Tuxemon/tuxemon/core/components/event/actions/npc.py", line 85, in create_npc
    npc = player.Npc(sprite_name=animations, slug=slug)
  File "/tmp/Tuxemon/tuxemon/core/components/player.py", line 720, in __init__
    npc_data = npcs['npc'][slug]
TypeError: 'JSONDatabase' object has no attribute '__getitem__'
```